src/main/webapp/resources/index.js

```diff
@@ -0,0 +1,77 @@
+$(document).ready(function () {
+
+    const productsListTemplate = Handlebars.compile($("#card-template").html());
+    //TODO finish when category logic will be added
+    const categoryListTemplate = Handlebars.compile($("#dropdown-category-template").html());
+
+    const limit = 6;
+    const keyStorage = "my-cart";
+
+    function addToCart(product) {
+        let newCart = JSON.parse(localStorage.getItem(keyStorage)) || {};
+
+        let productCounter = newCart[product.productId];
+
+        if (productCounter) {
+            productCounter.count++;
+        } else {
+            productCounter = {
+                count: 1,
+                product: product
+            };
+        }
+        newCart[product.productId] = productCounter;
+        localStorage.setItem(keyStorage, JSON.stringify(newCart));
+    }
+
+    // page is 1-indexed
+    function showPage(activePage) {
+        const searchString = $("#search-input").val();
+        const data = {
+            title: searchString,
+            offset: (activePage - 1) * limit,
+            limit: limit
+        };
+
+        $.get("/app/products", data, function (data, status, jqXHR) {
```

Comment:
The context part `/app` has to be extracted and defined globally since the context can be easily changed when redeploying.